volatility3/framework/contexts/__init__.py

```diff
@@ -245,7 +245,7 @@ def object(
         """
         if constants.BANG not in object_type:
             object_type = self.symbol_table_name + constants.BANG + object_type
-        else:
+        elif not object_type.startswith(self.symbol_table_name + constants.BANG):
```

Comment:
Yep, this is probably what it should always have been, sorry I didn't include it first time round!